docs/UserGuide.md

```diff
@@ -16,7 +16,24 @@ If you can type fast, WedLinker can get your contact management tasks done faste
 
 ## Quick start
 
-1. Ensure you have `Java 17` or above installed in your Computer. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
+1. Ensure you have `Java 17` or above installed on your computer by performing the following:
+   1. Open your system terminal
+      1. Windows: <br>
+         1. Hold down Windows button and R (⊞ Win + R ) 
+         2. Type `cmd` and press enter
+         
+      1. MacOS -- do one of the following: <br>
+          - Click the Launchpad icon in the Dock, type `Terminal` in the search field, then click Terminal, OR
+          - In the Finder, open the `/Applications/Utilities` folder, then double-click Terminal.
+   1. Type `java -version` and press enter
+   1. If you have `Java 17 installed, the terminal should look something like:
+   ```
+         C:\Users\UserName>java -version
+         java version "17.0.12" 2024-07-16 LTS
+         Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
+         Java HotSpot(TM) 64-Bit Server VM (build 17.0.12+8-LTS-286, mixed mode, sharing)
+    ```
+2. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
```

Comment:
Why is this 2. and the next step also 2.?

Reply:
![image](https://github.com/user-attachments/assets/410e812e-58d3-4675-a105-11dc7fdb926d)

GFMD/MD automatically numbers that's why there's "repeated" numbers. Having
```
1.
1.
1.
```
will make MD auto-number :D

Reply:
Oh yea i realised after i pulled from your branch... should have done that from the start oops sorry! then LGTM